docs/deployment-process/operations-runbooks/index.md

```diff
@@ -37,11 +37,8 @@ You can create many runbooks per project and share the [project variables](/docs
 
 ### Current Limitations
 
-**Prompted Variables**
-Prompted variables are not currently supported for the Runbooks feature, but we aim to have support for prompted variables in the near future. There is no workaround at this time.
-
 **Scoping**
-You cannot scope project variables to either a Deployment or Runbook Process currently, but we do aim to support this in the near future. A workaround is to namespace your Runbook-specific project variables. For example: Your project variable could be named `Runbook.Foo`. This naming prefix may help you easily see and consume "Runbook" variables in your Runbook Process.
+- You cannot scope project variables to a Deployment Process Action as well as a Runbook Process action currently, but we do aim to support this in the near future.
```

Comment:
Could we update the title of this to be "Scoping to Steps/Actions" so it's more clear this is all about actions. I think most customers think about steps, not actions, so we should use the word Steps here I think

Reply:
Did we want to change all occurrences to use step? i.e.
- You cannot scope project variables to a Deployment Process *step* as well as a Runbook Process *step* currently, but we do aim to support this in the near future.